utils/digitemp/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2009-2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+# Comment: This makefile is based on the oldpackage makefile by acinonyx / nbd
+# https://dev.openwrt.org/browser/packages/utils/digitemp/Makefile
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=digitemp
+PKG_VERSION:=3.7.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/cvbkf/digitemp.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=bb09dc5acb58dd2c076605e2204d96214f2afe5f
+PKG_MAINTAINER:=Christian Weinmueller <cw@127null01.com>
+PKG_LICENSE:=GPL-2.0+
+
+
```

Comment:
Drop this empty line, one empty line is sufficient.

Reply:
Ok will do